k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSettings.java

```diff
@@ -703,12 +710,21 @@ public boolean onPreferenceClick(Preference preference) {
             mCryptoSupportSignOnly = (CheckBoxPreference) findPreference(PREFERENCE_CRYPTO_SUPPORT_SIGN_ONLY);
 
             mCryptoApp.setValue(String.valueOf(mAccount.getCryptoApp()));
+            if (OpenPgpAppPreference.isApgInstalled(getApplicationContext())) {
+                mCryptoApp.addLegacyProvider(APG_PROVIDER_PLACEHOLDER, getString(R.string.apg), R.drawable.ic_apg_small);
```

Comment:
Did this but reverted, because the APG icon has a 1px black border that looks awful. I prefer to have this reliable without dropping the icon.